int64 id
float32 world_x_init
float32 world_y_init
int32 left_yaw_max
int32 right_yaw_max
